ci/build_container/build_recipes/luajit.sh

```diff
@@ -2,15 +2,21 @@
 
 set -e
 
-VERSION=2.1.0-beta3
-SHA256=409f7fe570d3c16558e594421c47bdd130238323c9d6fd6c83dedd2aaeb082a8
 if [[ "${OS}" == "Windows_NT" ]]; then
   exit 0
 fi
 
-curl https://github.com/LuaJIT/LuaJIT/archive/v"$VERSION".tar.gz -sLo LuaJIT-"$VERSION".tar.gz \
-  && echo "$SHA256" LuaJIT-"$VERSION".tar.gz | sha256sum --check
-tar xf LuaJIT-"$VERSION".tar.gz
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+$($SCRIPT_DIR/versions.py luajit)
+
+FILE_NAME=$(basename "$FILE_URL")
+
+curl "$FILE_URL" -sLo "$FILE_NAME" \
+  && echo "$FILE_SHA256" "$FILE_NAME" | sha256sum --check
+tar xf "$FILE_NAME"
```

Comment:
We could consider factoring out this common code between the scripts, but since we have only two build recipes left (and hopefully this number declines even further), probably not worth it.

Reply:
+1, IMO gperftools is very close to be moved to configure_make, and lua we will moved out as well.

Reply:
Yeah - I saw the gperf revert so I didn't bother refactoring this bit knowing changes are imminent.